### ISSUE TITLE
How to test? Please!!! I have got the pre-trained models， but i don't know the test commend。

### ISSUE THREAD

**Issue by Studyhao** [thstkdgus35/EDSR-PyTorch#300]

**Comment by Awenjie10** [comment]
hello, Have you learned about it? I got some mistakes about the testing, I used my own images, but the model outputs only artifacts.

**Comment by mylifeasazucchini** [comment]
Maybe I am too late but for anyone else, you need to add a few additional parameters and the function call should look like this:

python main.py --data_test Demo --scale 4 --pre_train ../models/EDSR_x4.pt --n_resblocks 32 --n_feats 256 --res_scale 0.1 --test_only --save_results

**Comment by haikunzhang95** [comment]
Thank you very much! It's works for me.

**Comment by ymtupup** [comment]
Why the --save_results   come out error?
![捕获1](https://user-images.githubusercontent.com/41365608/135191458-5065c935-0fa0-4411-868a-9fcc913e6358.PNG)

**Comment by Ralapfranco** [comment]
> n_resblocks 32 --n_feats 256 --res_scale 0.1
 which i use" n_resblocks 32 --n_feats 256 --res_scale 0.1"it is worse than that which i do not use the“ n_resblocks 32 --n_feats 256 --res_scale 0.1”

**Comment by liu1216** [comment]
> hello, Have you learned about it? I got some mistakes about the testing, I used my own images, but the model outputs only artifacts.


Excuse me, have you solved it? When I test with my own test set, I can only output super-resolution images, and PSNR = 0?

**Comment by Ralapfranco** [comment]
sorry! i'm too late. You should debug it！ i guess  that the module can't get the test path. 
















At 2022-08-30 16:39:18, "liu1216" ***@***.***> wrote:

hello, Have you learned about it? I got some mistakes about the testing, I used my own images, but the model outputs only artifacts.

Excuse me, have you solved it? When I test with my own test set, I can only output super-resolution images, and PSNR = 0?
